client/src/app/trash/trash.component.scss

```diff
@@ -12,15 +12,15 @@
   flex-grow: 1;
 }
 
-.note-card .mat-card-actions {
+.unpinned-note-card .mat-card-actions {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
 
 // For some reason, Angular gives margins to every button but the first.
 // So, we have to go back and add these margins manually.
-.note-card .mat-card-actions .mat-button:first-child {
+.unpinned-note-card .mat-card-actions .mat-button:first-child {
```

Comment:
Don't we have a `notes.scss` to do this in?

Reply:
Might be something to take note of for refactoring in the future

Reply:
If its global to multiple components then it should just be in `styles.scss`. But really if you are reusing cards like that the card itself with all its styles should be its own component with the data to display in it passed in.

Reply:
It's only used on owner page